warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+
+findJar (){
+  ls -1 ../../lib/$1-[0-9]*.jar | sort | tail -1
+}
+findFirstJar (){
+  ls -1 ../../lib/$1-[0-9]*.jar | sort | head -1
+}
+findAllJars (){
+  ls -1 ../../lib/$1-[0-9]*.jar | sort | paste -sd ':' -
+}
+findWebserviceJar (){
+  ls -1 ../../lib/$1[-0-9.]*jar | sort | head -1
+}
+findProvenanceJar (){
+  ls -1 ../../lib/$1-[0-9.]*.*.jar |  grep -v with-dependencies | sort | tail -1
+}
+
+
+
+CONF_DIR=../../config
+DATAWAVE_INGEST_CSV_JAR=$(findJar datawave-ingest-csv)
+DATAWAVE_INGEST_JSON_JAR=$(findJar datawave-ingest-json)
+DATAWAVE_INGEST_WIKIPEDIA_JAR=$(findJar datawave-ingest-wikipedia)
+DATAWAVE_INGEST_NYCTLC_JAR=$(findJar datawave-ingest-nyctlc)
+DATAWAVE_INGEST_CORE_JAR=$(findJar datawave-ingest-core)
+DATAWAVE_INGEST_CONFIG_JAR=$(findJar datawave-ingest-configuration)
+DATAWAVE_COMMON_JAR=$(findJar datawave-common)
+DATAWAVE_BALANCERS_JAR=$(findJar datawave-balancers)
+DATAWAVE_METRICS_CORE_JAR=$(findJar datawave-metrics-core)
+COMMON_UTIL_JAR=$(findWebserviceJar datawave-ws-common-util)
+COMMON_JAR=$(findWebserviceJar datawave-ws-common)
+DATAWAVE_CORE_JAR=$(findJar datawave-core)
+DATAWAVE_WS_QUERY_JAR=$(findWebserviceJar datawave-ws-query)
+DATAWAVE_WS_CLIENT_JAR=$(findWebserviceJar datawave-ws-client)
+CURATOR_FRAMEWORK_JAR=$(findJar curator-framework)
+CURATOR_UTILS_JAR=$(findJar curator-client)
+COMMONS_LANG_JAR=$(findJar commons-lang)
+COMMONS_LANG3_JAR=$(findJar commons-lang3)
+COMMONS_COLLECTIONS_JAR=$(findJar commons-collections4)
+COMMONS_CONFIGURATION_JAR=$(findJar commons-configuration)
+AVRO_JAR=$(findJar avro)
+HTTPCLIENT_JAR=$(findJar httpclient)
+HTTPCORE_JAR=$(findJar httpcore)
+COMMONS_IO_JAR=$(findJar commons-io)
+COMMONS_POOL_JAR=$(findJar commons-pool)
+COMMONS_JCI_CORE_JAR=$(findJar commons-jci-core)
+COMMONS_JCI_FAM_JAR=$(findJar commons-jci-fam)
+GUAVA_JAR=$(findJar guava)
```

Comment:
If this is replacing ingest-libs.sh, you will need:
CAFFEINE_JAR=$(findJar caffeine)

Reply:
Thanks for catching that.  I'm going to close this and create a new PR that accounts for those types of changes.